pghrep/templates/report.tpl

```diff
@@ -2,10 +2,10 @@ Current values
 ===
 {{.Current}}
 
-Recomended values
+Conclusions
 ===
-{{.Recommended}}
+{{.Conclusion}}
 
-Conclusions
+Recomended values
```

Comment:
"Recommended" (double `m`)

Reply:
Let's discuss how to name this section better (I had idea that "Recommendations" is the proper subtitle, but definitely we can discuss)

Reply:
Agree with @NikolayS